app/main.py

```diff
@@ -3,14 +3,15 @@
 from werkzeug.utils import secure_filename
 import os
 import shutil
-import uuid
+# import uuid
 import traceback
 import generate_csv_eventyay
 from svg_to_png import do_svg2png
 
 APP_ROOT = os.path.dirname(os.path.abspath(__file__))
 UPLOAD_FOLDER = os.path.join(APP_ROOT, 'static/uploads')
 BADGES_FOLDER = os.path.join(APP_ROOT, 'static/badges')
+SCRIPT = os.path.join(APP_ROOT, 'merge_badges.sh')
```

Comment:
@gabru-md All the files were converted to python.So i dont think we need this line

Reply:
okay!